styles/layout.css

```diff
@@ -0,0 +1,23 @@
+.filter {
+    display: inline-block;
+    padding: 10px;
+    width: auto;
+    font-size: 24px;
+}
+
+.content {
+    float: left;
+    min-width: 90%;
+}
+
+@media screen and (min-width: 400px) {
+    .filter {
+        display: block;
+        float: left;
+    }
+
+    article {
+        min-width: 90%;
+        float:left;
+    }
+}
```

Comment:
OK. I see that you [removed](https://github.com/suhk/code-blog/pull/6/files#diff-eacf331f0ffc35d4b482f1d15a887d3bL7) index.css from your index.html. It's confusing to include this index.css file in your master branch from this pull request since you don't seem to be actually be using it. Try and keep unused code out of your master branch to avoid confusion and clutter.